Result of commenting out par(top.par).

ldecicco-USGS opened the pull request.

ldecicco-USGS committed:
Result of commenting out par(top.par).

ldecicco-USGS committed:
Can't help myself! The xaxs/yaxs default was driving me insane.

ldecicco-USGS commented on inst/doc/gsplotIntro.Rmd:
````diff
@@ -114,7 +114,7 @@ For timeseries, it is sometimes helpful to plot data on the same graph to make c
 ```{r echo=TRUE, fig.cap="Fig. 4 Water temperature timeseries on primary y-axis with pH timeseries on secondary y-axis.", fig.width=6, fig.height=6}
 site <- '04193490'
-demoPlot <- gsplot(mar=c(7.1, 4.1, 4.1, 2.1)) %>% 
+demoPlot <- gsplot(mar=c(7.1, 4.1, 4.1, 4.1)) %>% 
````
This plot still shows that if 2 views share an axis, the axis call is being repeated. So, doesn't actual solve #248

ldecicco-USGS commented:
Fixes #109

jordansread commented on R/print.R:
```diff
@@ -42,7 +42,7 @@ print.gsplot <- function(x, ...){
   top.par <- par(no.readonly=T)
   for (i in which(names(views) %in% 'view')){
-    par(top.par)
+    # par(top.par)
```
I don't understand why this was the issue w/ layout. This is a loop within 1 gsplot object. layout was calling 3 or 4 different gsplot objects.

ldecicco-USGS replied:
Wrote this in the wrong place. I think it's because the layout function knows how to define par$usr

ldecicco-USGS commented:
Might be more to it, but my initial guess is that layout does some magic to change par$usr

jordansread commented:
Just so I understand this, the `par(top.par)` literally did nothing other than break `layout`? I worry that it was doing something that we don't actually test for...but I could be wrong.

ldecicco-USGS commented:
That's what I worry about too. As far as I know, and as far as what's being tested for, I haven't found anything that is now broke. But if anything jiggles your memory as to why it's in there, now's the time to think of it. I'll quick run through all the examples by hand and make sure they look like I think they should look.

jordansread commented:
:+1:

ldecicco-USGS committed:
Merging conflicts.

Merge branch 'master' of github.com:USGS-R/gsplot

Conflicts:
	README_files/figure-markdown_github/unnamed-chunk-2-1.png
	README_files/figure-markdown_github/unnamed-chunk-4-1.png
